packages/vscode-extension/src/chat/commands/nextstep/officeAddinSteps.ts

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CommandKey } from "../../../constants";
+import { CHAT_EXECUTE_COMMAND_ID } from "../../consts";
+import {
+  canOfficeAddInPreviewInLocalEnv,
+  isDebugSucceededAfterSourceCodeChanged,
+  isDependenciesInstalled,
+  isDidNoActionAfterScaffolded,
+  isFirstInstalled,
+  isHaveReadMe,
+  isPrequisitesCheckSucceeded,
+  isProjectOpened,
+} from "./condition";
+import { NextStep, WholeStatus } from "./types";
+
+// TODO: align the description with PM
+export const officeAddinSteps: () => NextStep[] = () => [
+  {
+    title: "Teams Toolkit",
+    description: `Teams Toolkit makes it simple to get started with app development for Microsoft Office Add-in using Visual Studio Code. You can start with a project template for a common custom app built for your org (LOB app) scenarios or from a sample. You can save setup time with automated app registration and configuration. You can run and debug your Office Add-in locally.`,
```

Comment:
Proposed: 
Teams Toolkit makes it simple to get started with app development for Microsoft Office Add-ins using Visual Studio Code.  Start with a sample or a project template for common custom app built for your org (LOB app) scenarios. Save setup time with automated app registration and configuration. You can run and debug your Office Add-in locally.